Makefile

```diff
@@ -34,4 +43,11 @@ release-checks:
 release: release-checks clean docs it
 	./release.sh $(release_version) $(next_version)
 
-.PHONY: clean docs test it it34 it35 it36 benchmark coverage release release-checks
+docker-it: nondocs-clean python-caches-clean
+	@if ! export | grep UID; then export UID=$(shell id -u); fi ; \
```

Comment:
I am not familiar with this syntax. Why is there an `@` in the first line?

Reply:
`@` causes `make` to suppress echoing: https://www.gnu.org/software/make/manual/make.html#Echoing

less junk on the output esp. with `if`'s that can confuse the user.

Reply:
Didn't know about that. I also saw now that this is actually just one long line separated by `\`, i.e. all output from this target will get suppressed.

Reply:
Just a small correction not all output will get suppressed (e.g. the output generated from the commands themselves e.g. `make it` output will get displayed as expected).

e.g. if you have a Makefile target called:

```
testme:
    @ls
```

this will display the `ls` output normally but not the `ls` command itself, which would be the case without `@`.

Reply:
Yes, I should have said "echoing" instead of "output". Thanks for the explanation.